src/scraper/stockx_guide.ts

```diff
@@ -0,0 +1,366 @@
+import axios from "axios";
+import { prisma, Int } from "../generated/prisma-client";
+import { totalmem } from "os";
+
+const headers = { headers: { "User-Agent": "Mozilla/5.0" } };
+const ModelBaseURL =
+  "https://stockx.com/api/browse?productCategory=sneakers&sort=release_date&order=DESC";
+const years: Array<string> = [
+  "2001",
+  "2002",
+  "2003",
+  "2004",
+  "2005",
+  "2006",
+  "2007",
+  "2008",
+  "2009",
+  "2010",
+  "2011",
+  "2012",
+  "2013",
+  "2014",
+  "2015",
+  "2016",
+  "2017",
+  "2018",
+  "2019"
+];
+const brands = [
+  {
+    name: "adidas",
+    models: ["yeezy", "ultra%20boost", "nmd", "iniki", "other"]
+  },
+  {
+    name: "air%20jordan",
+    models: [
+      "packs",
+      "spizike",
+      "one",
+      "two",
+      "three",
+      "four",
+      "five",
+      "six",
+      "seven",
+      "eight",
+      "nine",
+      "ten",
+      "eleven",
+      "twelve",
+      "thirteen",
+      "fourteen",
+      "fifteen",
+      "sixteen",
+      "seventeen",
+      "eighteen",
+      "nineteen",
+      "twenty",
+      "twenty-one",
+      "twenty-two",
+      "twenty-three",
+      "twenty-eight",
+      "twenty-nine",
+      "thirty",
+      "thirty-one",
+      "thirty-two",
+      "thirty-three",
+      "other"
+    ]
+  },
+  {
+    name: "nike",
+    models: [
+      "foamposite",
+      "kd",
+      "kobe",
+      "lebron",
+      "air%20force",
+      "air%20max",
+      "nike%20basketball",
+      "nike%20sb",
+      "other"
+    ]
+  },
+  {
+    name: "asics",
+    models: ["asics"]
+  },
+  {
+    name: "diadora",
+    models: ["diadora"]
+  },
+  {
+    name: "li%20ning",
+    models: ["li%20ning"]
+  },
+  {
+    name: "louis%20vuitton",
+    models: ["louis%20vuitton"]
+  },
+  {
+    name: "new%20balance",
+    models: ["new%20balance"]
+  },
+  {
+    name: "puma",
+    models: ["puma"]
+  },
+  {
+    name: "reebok",
+    models: ["reebok"]
+  },
+  {
+    name: "saucony",
+    models: ["saucony"]
+  },
+  {
+    name: "under%20armour",
+    models: ["under%20armour"]
+  }
+];
+
+/*
+ * @ Class StockX
+ * @ method run() - this function is for test
+ *          collectAll() - collect new products
+ *          createUrl() - create URL using url queries
+ *          seperateUrlByYears() - if there is too many product in one category,
+ *                                 Detail it
+ *          collectProductsInPage() - Get product in a page using URL
+ *                                    It start from page 1 to last page
+ *          insert Product() - insert each product infromation to DB
+ **/
+class StockX {
+  run = async () => {
+    const noUrls: Array<string> = [];
+    console.log("run");
+    const a = await this.collectAll();
```

Comment:
a, b, c 등 의미 없는 변수는 절대로 사용하지 않는게 좋습니다.
변수명으로 포함되는 데이터를 유추할 수 있는 이름을 사용해야 합니다.
```suggestion
    const initialUrls = await this.collectAll();
```